### PR TITLE
Feature/marketing test enrique

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java [hunk 1]
```diff
@@ -18,7 +18,12 @@
      *
      * @param leads contains a list value.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java]: You can improve this comment to "list of lead objects"

File: src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java [hunk 2]
```diff
@@ -27,4 +32,16 @@
      * @return int value.
      */
     public abstract List<Lead> getLeadsUpdated(List<Lead> leads);
+
+    /**
+     * Clicks on a button.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java]: update this comment

File: src/main/java/salesforce/ui/pages/leadlist/ChangeStatusLeadsClassicPage.java [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+
+    @Override
+    public void selectStatusLeads(final String statusLeads) {
+        Select select = new Select(leadStatusListBox);
+        select.selectByVisibleText(statusLeads);
+        clickOnSaveButton();
+    }
 
+    /**
+     * Clicks on a button.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/ChangeStatusLeadsClassicPage.java]: Clicks on saves button

File: src/main/java/salesforce/ui/pages/leadlist/LeadListClassicPage.java [hunk 4]
```diff
@@ -1,7 +1,13 @@
 package salesforce.ui.pages.leadlist;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
 import salesforce.entities.Lead;
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/LeadListClassicPage.java]: delete this line please

File: src/main/java/salesforce/ui/pages/leadlist/LeadListClassicPage.java [hunk 5]
```diff
@@ -11,18 +17,51 @@
  * @version 1.0 29 March 2020.
  */
 public class LeadListClassicPage extends AbstractLeadListPage {
-    @Override
-    public void changeLeadStatus(final List<Lead> leads) {
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/LeadListClassicPage.java]: delete this line please

File: src/test/resources/features/Lead.feature [hunk 6]
```diff
@@ -8,4 +8,3 @@ Feature: Lead
     Then the application should display this message in Leads Page only for Lightning Experience
       | Status was updated for 3 Leads. |
       And Leads Page should display the leads modified
-
```
Comment (JuanCarlosMartinez4) [inline src/test/resources/features/Lead.feature:11]: add a new line here please

File: src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java (hunk 2 above)
Comment (Enrique-C) [reply src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java]: It clicks on a button.

File: src/main/java/salesforce/ui/pages/leadlist/ChangeStatusLeadsClassicPage.java (hunk 3 above)
Comment (Enrique-C) [reply src/main/java/salesforce/ui/pages/leadlist/ChangeStatusLeadsClassicPage.java]: the name method says that

File: src/test/resources/features/Lead.feature [hunk 7]
```diff
@@ -8,4 +8,5 @@ Feature: Lead
     Then the application should display this message in Leads Page only for Lightning Experience
       | Status was updated for 3 Leads. |
       And Leads Page should display the leads modified
-     
\ No newline at end of file
+
```
Comment (JuanCarlosMartinez4) [inline src/test/resources/features/Lead.feature]: delete this line, it is only one empty line at end of file

File: src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java [hunk 8]
```diff
@@ -27,4 +32,16 @@
      * @return int value.
      */
     public abstract List<Lead> getLeadsUpdated(List<Lead> leads);
+
+    /**
+     * Clicks on a button "Change Status".
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/leadlist/AbstractLeadListPage.java]: delete double quotes, that is no the correct way the documentation

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**